DuckDuckGo/OmniBar.swift

```diff
@@ -276,7 +276,7 @@ class OmniBar: UIView {
         }
 
         customIconView.isHidden = true
-        privacyInfoContainer.privacyIcon.isHidden = false
+        privacyInfoContainer.privacyIcon.isHidden = privacyInfo.isSpecialErrorPageVisible
```

Comment:
I saw there are no tests for the omnibar… but we want to get better at it I think if you make a OmniBar test class and just test this updatePrivacyIcon function will encourage people to add tests